app/models/pxe_server.rb

```diff
@@ -213,4 +213,21 @@ def ensure_menu_list(menu_list)
   def self.display_name(number = 1)
     n_('PXE Server', 'PXE Servers', number)
   end
+
+  def self.verify_depot_settings_queue(userid, options)
+    task_opts = {
+      :action => "Verify PXE Server Credentials",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => name,
+      :method_name => "verify_depot_settings",
+      :queue_name  => "generic",
+      :role        => "ems_operations",
```

Comment:
I'm not sure generic queue + ems_operations role will work.  @agrare @bdunne ?

Reply:
PXE servers are not tied to an EMS